tests/test_create_gmshIO.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from gmsh_utils.gmsh_IO import GmshIO
+import numpy as np
+import pytest
```

Comment:
first import build-in python packages, then-> installed packages -> then project imports.

import pytest
import numpy as np

from gmsh_utils.gmsh_IO import GmshIO

 Also pytest is imported twice :D.

Reply:
:D this test is indeed no longer needed. I'll remove it.